tools/wptrunner/wptrunner/browsers/edge.py

```diff
@@ -84,6 +85,12 @@ def start(self, **kwargs):
 
     def stop(self, force=False):
         self.server.stop(force=force)
+        time.sleep(3)
```

Comment:
Can we avoid waiting this long by polling anything? Are we waiting for some process to terminate (the WebDriver server)? Do we need to wait after calling `taskkill.exe` below too?

Reply:
I can keep polling here(every 500 ms?) for tasklist to have no MicrosoftEdge.exe processes running. I expect that server exe exits pretty quickly and we're waiting on Edge browser windows to exit.

Reply:
We can poll pretty quickly depending on what order of magnitude we expect it to take, really. You'd probably have to just try and see how long it generally takes for them all to exit.

Reply:
Right now I don't see them exiting at all, so we end up terminating Edge processes after every test. Once we have proper fix in place, then I'll be able to test and see how long it usually takes for process to exit.

Reply:
@gsnedders I've updated the wait logic to reduce wait time and only wait if we see driver process still running. If we don't see driver process running, then we assume it already got closed after issuing close command to browser and we terminate any remaining browser instances.